.vscode/settings.json

```diff
@@ -10,7 +10,7 @@
 		"tgui/.pnp.*": true
 	},
 	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": true
+		"source.fixAll.eslint": "explicit"
```

Comment:
this shouldnt be here. this gets added by vsc every time and needs to manually changed back/uncommitted